README.md

```diff
@@ -1 +1,241 @@
-# pino-test
\ No newline at end of file
+# pino-test
+
+A set of API to easily test the Pino logger.
+
+## Getting started
+```sh
+npm install pino-test --save-dev
+```
+
+```js
+const test = require('node:test')
+const pino = require('pino')
+const pinoTest = require('pino-test')
+
+test('pino should log a info message', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected)
+})
+```
+
+Using tap
+```js
+const pino = require('pino')
+const pinoTest = require('pino-test')
+
+test('pino should log a info message on tap', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected)
+})
+
+// with a own assert function
+test('pino should log a info message using a own assert function', async ({ same }) => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected, same)
+})
+```
+
+Using jest
+
+```js
+const pino = require('pino')
+const pinoTest = require('pino-test')
+
+test('pino should log a info message on jest', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+  logger.info('hi world')
+
+  const expected = [
+    { msg: 'hello world', level: 30 },
+    { msg: 'hi world', level: 30 }
+  ]
+  await pinoTest.consecutive(stream, expected)
+})
+
+// with a own assert function
+function is (received, expected, msg) {
+  expect(received).toStrictEqual(expected)
+}
+
+test('pino should log a info message using a own assert function on jest', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+  logger.info('hi world')
+
+  const expected = [
+    { msg: 'hello world', level: 30 },
+    { msg: 'hi world', level: 30 }
+  ]
+  await pinoTest.consecutive(stream, expected, is)
+})
+```
```

Comment:
This whole block is redundant. It doesn't seem to show anything unique about using `jest` over `tap` or `node:test`.